src/widget/woverview.cpp

```diff
@@ -95,8 +95,10 @@ void WOverview::setup(const QDomNode& node, const SkinContext& context) {
 
     for (int i = 0; i < m_marks.size(); ++i) {
         const WaveformMarkPointer& mark = m_marks[i];
-        if (mark->m_pPointCos) {
-            mark->m_pPointCos->connectValueChanged(this,
+        if (mark->isValid()) {
+            //mark->m_pPointCos->connectValueChanged(this,
```

Comment:
Please delete code that is commented out. Nobody will know if this is old code that is unused now or if this is new code that is supposed to be used sometimes in the future.